platform/features/timeline/bundle.js

```diff
@@ -182,6 +182,16 @@ define([
                                 "capacity"
                             ],
                             "pattern": "^-?\\d+(\\.\\d*)?$"
+                        },
+                        {
+                            "name": "Battery Starting SOC",
+                            "control": "textfield",
+                            "required": false,
+                            "conversion": "number",
+                            "property": [
+                                "startingSOC"
+                            ],
+                            "pattern": "^-?\\d+(\\.\\d*)?$"
```

Comment:
Sounds like percentages are [slightly preferred](https://github.com/nasa/openmct/issues/1185#issuecomment-248102430) (thanks @RCarvalhoNASA!), which conveniently means we can dodge those validation limitations (for now)